### DISCUSSION TITLE
Is Evennia alive? Is anyone out there?

### DISCUSSION THREAD

**Discussion by Gnomic00** [evennia/evennia#3610] [Evennia Questions]
I have a few questions :

1.  Is this discussion board still used or has everyone moved elsewhere?
2. Is Evennia still active and maintained?
3. If so, will newer versions of Django, Sphinx, etc. be supported? Is there a roadmap for this?
4. Is there a list of tasks that need to be done? Where can I find it? 

I have more questions but let’s see if there are answers to these.

**Comment by volundmush** [comment]
1. Well, getting a response should answer this one.
2. Yes. Check out recent commits to the main branch if you want a peek at how active it's been!
3. I don't have authoritative answers to this, but we do update dependencies at a steady pace. Django's been updated before, it probably will be again.
4. The issues tab on Github! Lots of stuff to go over there. Sometimes there's milestones/roadmaps posted on Github...

I highly recommend joining the Evennia Discord. Lots more discussion happens there.

**Reply by MurakumoXGP** [reply]
Holy Hell Volund? I havent seen you in years.

**Reply by volundmush** [reply]
I'm around. Just rarely in Evennia territory. Still a member on the Discord though. Feel free to DM me anytime.

**Comment by Griatch** [comment]
Volund already answered, but for a more 'official' answer: 

1.  These discussions are used, but as it has turned out, it has more been for long-term discussions, sharing code snippets and announcing updates. Most day-to-day discussion happens on the discord server (see https://github.com/evennia/evennia/discussions/2267 for links). 
2. Yes.
3. We update dependencies as we go, but we generally don't keep up to date with the bleeding edge since we also maintain backwards compatibility for a while (major Django versions sometimes introduce breaking changes we need to consider). So we are upgrading, but slower. If there is a particular Django feature you need, consider making a ticket. Sphinx is tougher; we are on a pretty old version for now because it's a lot of work to update the build system (we have to do a lot of custom code to build Sphinx in markdown with full code coverage, so upgrading sphinx is more than just a version bump).  
4. Plenty of issues to help out with in the github issue tracker. Read more about getting and giving help [here](https://www.evennia.com/docs/latest/Contributing.html).
